integration-tests/environment-helpers/bootstrap-app.js

```diff
@@ -2,30 +2,60 @@ const path = require("path")
 const express = require("express")
 const getPort = require("get-port")
 const { isObject } = require("@medusajs/utils")
+const { setContainer } = require("./use-container")
+const { setPort, setExpressServer } = require("./use-api")
 
-module.exports = {
-  bootstrapApp: async ({ cwd, env = {} } = {}) => {
-    const app = express()
+async function bootstrapApp({ cwd, env = {} } = {}) {
+  const app = express()
 
-    if (isObject(env)) {
-      Object.entries(env).forEach(([k, v]) => (process.env[k] = v))
-    }
+  if (isObject(env)) {
+    Object.entries(env).forEach(([k, v]) => (process.env[k] = v))
+  }
 
-    const loaders = require("@medusajs/medusa/dist/loaders").default
+  const loaders = require("@medusajs/medusa/dist/loaders").default
 
-    const { container, dbConnection } = await loaders({
-      directory: path.resolve(cwd || process.cwd()),
-      expressApp: app,
-      isTest: false,
-    })
+  const { container, dbConnection } = await loaders({
+    directory: path.resolve(cwd || process.cwd()),
+    expressApp: app,
+    isTest: false,
+  })
 
-    const PORT = await getPort()
+  const PORT = await getPort()
 
-    return {
-      container,
-      db: dbConnection,
-      app,
-      port: PORT,
+  return {
+    container,
+    db: dbConnection,
+    app,
+    port: PORT,
+  }
+}
+
+module.exports = {
+  bootstrapApp,
+  startBootstrapApp: async ({
+    cwd,
+    env = {},
+    skipExpressListen = false,
```

Comment:
How do we handle `verbose: true` here?

Reply:
Verbose was used for the separate process to redirect the output to the main process console here it is on the same process, so you already have the output

Reply:
it isn't another process. logs, etc will be shown by default.

Reply:
Thanks! 🍻